### PR TITLE
Updated header styling for subpages

### PR DESCRIPTION
**Pull request by hn4733**
The following has been done: 
1. Styled the header for the subpages. 
2. Homepage has site identity (site title + blue ring) without styled menu items.
3. Optimised the header for mobile view -> check subpage header.
4. Styling of header added in /src/styles/header-styles.js

### CODE REVIEW

File: .codesandbox/workspace.json [hunk 1]
```diff
@@ -0,0 +1,41 @@
+{
```
Comment (fridaland) [inline .codesandbox/workspace.json]: Interesting, do we need to commit this file?

File: src/components/molecules/navigation.js [hunk 2]
```diff
@@ -21,16 +21,15 @@ export default function Navigation() {
...
-        <li key={link.name}>
-          <Styled.a
-            as={Link}
-            to={link.url}
-          >
-            {link.name}
-          </Styled.a>
-        </li>
-      ))}
+      <ul>
```
Comment (fridaland) [inline src/components/molecules/navigation.js:24]: Nice!

File: src/components/organisms/navbar.js [hunk 3]
```diff
@@ -1,19 +1,39 @@
...
+        <div></div>
+        <div></div>
+      </button>
+    </Header>
+  )
+}
 
 NavBar.propTypes = {
-  siteTitle: PropTypes.string,
+  siteTitle: PropTypes.string
```
Comment (fridaland) [inline src/components/organisms/navbar.js]: We are in favor of trailing commas here :) 
```suggestion
  siteTitle: PropTypes.string,
```

File: src/components/organisms/navbar.js [hunk 4]
```diff
@@ -1,19 +1,39 @@
...
+import blueRingImage from "../../images/logos/ring-blue.png"
 
-const NavBar = () => (
-  <header>
-    <Navigation />
-  </header>
-)
+const NavBar = ({ isHome }) => {
+  const [hamburgerState, setHamburgerState] = useState(false)
+  console.log({ hamburgerState })
```
Comment (fridaland) [inline src/components/organisms/navbar.js]: Thoughts on removing this `console.log`?

**Review by fridaland — CHANGES_REQUESTED** [review]
This is awesome! Thanks for your contribution! Just have a couple minor thoughts/comments.

**Review by hn4733 — COMMENTED** [review]
Updated the navBar.js, and layout.js files
1. Type checking added for the "isHome" prop.
2. Removed console.log in navBar.js
3. Added trailing comma for the propTypes.
4. Removed the codesandbox workspace file (pushed with the commit from codesandbox.io)

File: src/components/organisms/navbar.js (hunk 4 above)
Comment (hn4733) [reply src/components/organisms/navbar.js]: I have removed the console log in the review file now. :)

File: .codesandbox/workspace.json (hunk 1 above)
Comment (hn4733) [reply .codesandbox/workspace.json]: No worries :) happy to help. I have removed this file as part of the new review. Please check the latest updates.

**Review by fridaland — APPROVED** [review]

File: src/components/organisms/navbar.js [hunk 5]
```diff
@@ -1,19 +1,41 @@
...
+        <div></div>
+        <div></div>
+      </button>
+    </Header>
+  )
+}
 
 NavBar.propTypes = {
   siteTitle: PropTypes.string,
+  isHome: PropTypes.Bool,
```
Comment (fridaland) [inline src/components/organisms/navbar.js]: There are a couple issues with the build on this Proptype, looks like it just needed to be lowercase!
```suggestion
  isHome: PropTypes.bool,
```

File: src/components/templates/layout.js [hunk 6]
```diff
@@ -14,9 +14,15 @@ const Layout = () => {
...
   `)
 
-  return (
-    <NavBar siteTitle={data.site.siteMetadata.title} />
-  )
+  return <NavBar isHome={isHome} siteTitle={data.site.siteMetadata.title} />
+}
+  
+Layout.propTypes = {
+  isHome: PropTypes.Bool,
```
Comment (fridaland) [inline src/components/templates/layout.js]:
```suggestion
  isHome: PropTypes.bool,
```
Also looks like we forgot to import : 
```
import PropTypes from "prop-types"
```

**Review by fridaland — CHANGES_REQUESTED** [review]
A few minor comments after checking this out locally

**Comment by zopa** [comment]
One thing --- the big In The Fight in the header shouldn't wrap. ie, 
image

rather than

header-wrap

**Comment by hn4733** [comment]
I've reviewed your feedback and adjusted accordingly. Also adjusted doubles quotes to single for the import statements in navbar.js and layout.js. Tested everything to work correctly. However, looking at the console logs, I have noticed that there's quite a few errors that have not been addressed ie. no key values placed in loops and some data that are undefined. You may need to create issues for those. I'm not sure what @zopa is referring to but the header looks correct on all subpages - homepage header was originally kept with no styling - brief was to only style the **subpage** header.

**Comment by fridaland** [comment]
@hn4733 Great! Thanks for those updates, I believe what @zopa was referring to is that the actual `IN THE FIGHT` words should not span to two lines. The screenshot above was taken when we checked out the branch locally, so to adjust we would just have the title span one line and the logo sized appropriately next to it. Let me know if this makes sense and if you are able to adjust!

_***Edit:_ looks like the style issue with the title is sorted out with the updates!

 Please make sure to run `eslint` and resolve all lints now that we have incorporated it into our CI. :)

**Review by fridaland — APPROVED** [review]
Looks like those updates sorted out the issues with the logo ✨ 
Thanks for your contribution! As soon as CI passes (looks like just a few minor lints) this is good to go!

Tip: some of these may be resolved by running `eslint --fix src`
To run the linter: `eslint src`

**Comment by fridaland** [comment]
Took care of those lints for ya. Great job @hn4733!

**Comment by hn4733** [comment]
> Took care of those lints for ya. Great job @hn4733!

Thanks :) Glad I could contribute!

**Pull request merged**